frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/initialize-vm.ts

```diff
@@ -38,8 +39,9 @@ const resolveDataVolumeName = (
     !isTemplate || isPlainDataVolume ? newName : VM_TEMPLATE_NAME_PARAMETER,
   );
   return dataVolumeWrapper.getType() === DataVolumeSourceType.PVC &&
```

Comment:
I tried shortening this since the first two `if` statements both result in using the existing name. Let me know if my implementation is insufficient.

Reply:
I wonder, maybe we should simplify it a bit more and use suffix indiscrimintallely in creation:

```
if isTemplate and is not plainDataVolume
  use just the name
else
  use name (injected with vmName) and append random suffix 
```

Reply:
Sweet and simple. I like it.